test/Linter/Yaml/YamlLinterTest.php

```diff
@@ -92,4 +161,19 @@ function provideYamlValidation()
             ['fixture' => 'invalid.yml', 'errors' => 1],
         ];
     }
+
+    private function hasSymfonyLinterVersion($version, $operator = null)
```

Comment:
do not test against version, test against features.what if feature would be backported? what if feature would be removed in 5.0 ? when pinning to version, you need to observe and maintain it. when pinning to features (which you are already observing), you don't care in which version they exist or not